pom.xml

```diff
@@ -171,6 +169,49 @@
     <vers.jcommander>1.35</vers.jcommander>
     <vers.spark>${cdh.spark.version}</vers.spark>
     <vers.snappy>1.0.5.3</vers.snappy>
+    
+    <!-- HDP Versions -->
+    <vers.hdp.avro-base>1.7.6</vers.hdp.avro-base> <!-- The version CDH is based on -->
+    <vers.hdp.avro>${cdh.avro.version}</vers.hdp.avro>
```

Comment:
`cdh.avro.version` is `1.7.6-cdh5.4.0`, but the line above sets avro-base to 1.7.6. Should they match?